http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -445,4 +450,14 @@ protected static boolean checkCachedAccountMatchesRequest(OidcAccount account, O
         return true;
     }
 
+    public static String getCryptographicValue(final String src) {
+        try {
+            MessageDigest md = MessageDigest.getInstance(SHA256);
+            md.update(src.getBytes());
+            return new String(md.digest(), StandardCharsets.UTF_8);
```

Comment:
Here, I think we could use ByteIterator.ofBytes(md.digest()).base64Encode(BASE64_URL, false).drainToString()

Reply:
done